### ISSUE TITLE
add make_grid or make_tiles

### ISSUE THREAD

**Issue by edgarriba** [kornia/kornia#2169]
We could add this to kornia directly, as make_grid or make_tiles

_Originally posted by @edgarriba in https://github.com/kornia/limbus/pull/15#discussion_r1082140602_

**Comment by lferraz** [comment]
Inputs:
images (Union[torch.Tensor, List[torch.Tensor]]): 4D Tensor with shape (B x [1, 3] x H x W) or a list of tensors with the same shape ([1, 3] x H x W) or (H x W).
nrow (Optional[int], optional): Number of images in each row. Default: None -> sqrt(len(images)).

Outputs:
grid (torch.Tensor)

Code:
        # inspired by the images() function in visdom.__init__.py.
        if isinstance(images, list):
            # NOTE that stack adds a dim to the result even if there was a single image.
            # so, if images is [(Hx W)] then the stack shape is (1 x H x W)
            images = torch.stack(images, 0)
            if images.ndim == 3:
                # if images are (H x W) convert tensor to (B x 1 x H x W)
                images = images.unsqueeze(1)
        else:
            # in this case we will assume a single image was passed.
            if images.ndim == 3:
                images = images.unsqueeze(0)

        # convert tensor shape from (B x C x H x W) to a grid with shape (C x H' x W')
        if nrow is None:
            nrow = math.ceil(math.sqrt(images.shape[0]))
        elif nrow > images.shape[0]:
            nrow = images.shape[0]
        ncol: int = math.ceil(images.shape[0] / nrow)
        padding: int = 4
        grid: torch.Tensor = torch.zeros((images.shape[1],
                                          ncol * (images.shape[2] + padding) - padding,
                                          nrow * (images.shape[3] + padding) - padding)).to(images)
        j: int = 0
        h: int = images.shape[2] + padding
        w: int = images.shape[3] + padding
        for idx in range(images.shape[0]):
            i = idx % nrow
            j = idx // nrow
            grid[:, j * h:j * h + images.shape[2], i * w:i * w + images.shape[3]] = images[idx]
